addons/website_sale_delivery/tests/test_controller.py

```diff
@@ -56,3 +59,59 @@ def test_address_states(self):
 
         country_info = self.Controller.country_infos(country=US, mode="shipping")
         self.assertEqual(len(country_info['states']), 0)
+
+    def test_available_methods(self):
+        self.env['delivery.carrier'].search([]).action_archive()
+        self.product_delivery_poste = self.env['product.product'].create({
+            'name': 'The Poste',
+            'type': 'service',
+            'categ_id': self.env.ref('delivery.product_category_deliveries').id,
+            'sale_ok': False,
+            'purchase_ok': False,
+            'list_price': 20.0,
+        })
+        self.env['delivery.carrier'].create([
+            {
+                'name': 'Over 300',
+                'fixed_price': 20.0,
```

Comment:
I don't think it's necessary since it is computed from `self.product_id.list_price` (and is not used if `base_on_rule`)
```suggestion
```